css/main.css

```diff
@@ -416,6 +434,39 @@ textarea {
     brightness(92%) contrast(82%);
 }
 
+/* mobile menu */
+.menu-visible {
+  position: fixed;
+  display: block;
+  background-color: #e5e5e5;
+  height: 100%;
+  width: 100%;
+  top: 0;
+  padding: 0;
+}
```

Comment:
- [ ] I can see that the hamburger background is not covering the full height of the mobile menu. Also, kindly checkout the font size of the menus and their padding. 



| Your Design  | Figma Design |
| ------------- | ------------- |
| ![Your design](https://user-images.githubusercontent.com/63473584/175324384-26446265-264a-4926-8ad8-af10348f3f51.png)  | ![Figma Design](https://user-images.githubusercontent.com/63473584/175325156-f4b33211-bd02-4907-87e0-252471f493fb.png)  |